README.md

```diff
@@ -2,34 +2,26 @@
 
 # capstone-project
 
-> In this capstone project, I built an online website based on [CC Global Summit 2015](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) by Cindy Shin and [culture days' website page](https://culturedays.ca/en) that helped me to personalize the content.
+> In this capstone project, I built an online website based on [CC Global Summit 2015](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) by Cindy Shin.
 
 ## Built With
 
-- semantic HTML tags.
-- Apply best practices in HTML code.
-- Use CSS selectors correctly.
-- Use CSS box model.
-- Use Flexbox to place elements in the page.
-- Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
-- Use GitHub Pages to deploy web pages.
-- Apply JavaScript best practices and language style guides in code.
-- Use JavaScript to manipulate DOM elements.
-- Use JavaScript events.
-- Use objects to store and access data.
-- Communicate technical concepts to other technical people.
+- HTML
+- CSS
+- JAVASCRIPT
+- GitHub pages
 
 ## Links
 
 - [capstone project](https://vvoo21.github.io/capstone-project/)
+- [Project documentation](https://www.loom.com/share/583aebc197cd4b2c9ca06a5192bc1730)
```

Comment:
- Kindly add another video including the required content. It's a project requirement.
```
- The video should present the features of the project.
- It should have a maximum length of 5 minutes.
- The video should contain a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- The video should contain a demo of the project, showing the pages in different screen sizes.
- The video should contain a highlight about a piece of the code you find interesting or is proud of making.```